.gitlab-ci.yml

```diff
@@ -697,14 +705,8 @@ test_integ_qemux86_64_uefi_grub:
     reports:
       junit: results_integ_qemux86_64_uefi_grub.xml
   only:
-    # The variables expressions will work after GitLab release 12.0,
-    # see https://gitlab.com/gitlab-org/gitlab-ce/merge_requests/27925
-    # Meanwhile split them into two different conditions as a workaround
     variables:
       - $RUN_INTEGRATION_TESTS == "true"
-    variables:
-      - $TEST_QEMUX86_64_UEFI_GRUB == "true"
-      - $TEST_VEXPRESS_QEMU == ""
```

Comment:
I think my original intention here was to allow running integ tests with vexpress client using a combination of these variables.

Anyway, I think your change here made it cleaner.